internal/rhcs/cloudprovider/cloud_providers_state.go

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudprovider
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/terraform-redhat/terraform-provider-rhcs/internal/rhcs/common"
+)
+
+func cloudProviderSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"search": {
+			Description: "Search criteria.",
+			Type:        schema.TypeString,
+			Optional:    true,
+		},
+		"order": {
+			Description: "Order criteria.",
+			Type:        schema.TypeString,
+			Optional:    true,
+		},
+		"item": {
+			Description: "Content of the list when there is exactly one item.",
+			Type:        schema.TypeList,
+			Elem: &schema.Resource{
+				Schema: itemAttributes(),
+			},
+			Computed: true,
+		},
+		"items": {
+			Description: "Content of the list.",
+			Type:        schema.TypeList,
+			Elem: &schema.Resource{
+				Schema: itemAttributes(),
+			},
+			Computed: true,
+		},
+	}
+}
+func itemAttributes() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"id": {
+			Description: "Unique identifier of the cloud clusterservice. This is what " +
```

Comment:
this seems like a really weird replace-all. it should be 
```suggestion
			Description: "Unique identifier of the cloud provider. This is what " +
```

Reply:
handled via https://github.com/terraform-redhat/terraform-provider-rhcs/pull/263